app/controllers/roles_controller.rb

```diff
@@ -17,4 +11,13 @@ def update
     p params
     return render json: { test: 'hoho' }
   end
+
+  private
+
+  def user_roles
+    existing = current_user.roles
+    remainder = (1..(3-existing.count)).map{ |_| Role.new }
```

Comment:
Seeing these mad DJ mutiny golfing skillz brings me back to the good ol days :crying_cat_face: